zuul-core/src/main/java/com/netflix/zuul/filters/endpoint/ProxyEndpoint.java

```diff
@@ -156,6 +159,13 @@
 
     private static final CachedDynamicLongProperty MAX_OUTBOUND_READ_TIMEOUT_MS =
             new CachedDynamicLongProperty("zuul.origin.readtimeout.max", Duration.ofSeconds(90).toMillis());
+    /**
+     * Indicates how long Zuul should remember throttle events for an origin.  As of this writing, throttling is used
+     * to decide to cache request bodies.
+     */
+    private static final CachedDynamicLongProperty THROTTLE_MEMORY_SECONDS =
+            new CachedDynamicLongProperty("zuul.proxy.throtle_memory_seconds", Duration.ofMinutes(5).getSeconds());
```

Comment:
Typo on "throttle". Also, I think 5 minutes is too long of a threshold but can't argue with the idea that we would want to be conservative in estimating this.

Reply:
Fixed.   We can easily change this later.